js_modules/dagster-ui/packages/ui-core/src/app/useVisibleFeatureFlagRows.oss.tsx

```diff
@@ -27,4 +27,8 @@ export const useVisibleFeatureFlagRows = (): FeatureFlagRow[] => [
     key: 'Disable automatically loading default config in launchpad',
     flagType: FeatureFlag.flagDisableAutoLoadDefaults,
   },
+  {
+    key: 'Revert to separate asset graph groups per code location',
+    flagType: FeatureFlag.flagAssetGraphGroupsPerCodeLocation,
```

Comment:
Is this flagged just for the purposes of testing whether this is something users might reject? Or is this definitely the way we want the feature to behave?

Reply:
I think that's right - I'll confirm with Josh in Slack